### PR TITLE
Use prepend for inserts

### PR DESCRIPTION
**Pull request by mwichmann**
Scratching an itch: a number of places "append to the beginning", a better word to use here is "prepend".  This is a documentation change only, no executable statements are affected.

### CODE REVIEW

File: src/engine/SCons/Tool/fortran.xml [hunk 1]
```diff
@@ -250,7 +251,7 @@ through the automatically-generated
 &cv-link-_FORTRANINCFLAGS;
 construction variable,
 which is constructed by
-appending the values of the
+respectiviely prepending and appending the values of the
```
Comment (bdbaddog) [inline src/engine/SCons/Tool/fortran.xml]: typo in respectively

File: src/engine/SCons/Tool/swig.xml [hunk 2]
```diff
@@ -153,7 +153,8 @@ An automatically-generated construction variable
 containing the SWIG command-line options
 for specifying directories to be searched for included files.
 The value of &cv-_SWIGINCFLAGS; is created
-by appending &cv-SWIGINCPREFIX; and &cv-SWIGINCSUFFIX;
+by respectively prepending and appending
```
Comment (bdbaddog) [inline src/engine/SCons/Tool/swig.xml]: typo in respectively

File: src/engine/SCons/Tool/swig.xml [hunk 3]
```diff
@@ -238,7 +239,7 @@ through the automatically-generated
 &cv-_SWIGINCFLAGS;
 construction variable,
 which is constructed by
-appending the values of the
+respectively prepending and appending the values of the
```
Comment (bdbaddog) [inline src/engine/SCons/Tool/swig.xml]: typo in respectively

File: src/engine/SCons/Tool/fortran.xml (hunk 1 above)
Comment (mwichmann) [reply src/engine/SCons/Tool/fortran.xml]: sigh

**Comment by bdbaddog** [comment]
There is a python tool which will check for misspellings in source code (and includes many different file formats.  I'll have to dig it up and run a pass on SCons...

**Pull request merged**